### ISSUE TITLE
Add ability to ignore some tests

### ISSUE THREAD

**Issue by HugoGiraudel** [ffoodd/a11y.css#113]
I can see a scenario where a user is well aware of something but would like to ignore a given test, for debugging purpose or whatever.

What about providing a way to disable some tests when using the Sass API? Would be fairly easy to implement I must say. I can think of something like this:

```scss
$disabled-tests: (
  "advices": ("target-blank")
);
```

**Comment by ffoodd** [comment]
Well, this is a good idea: I'm often embarrassed by some tests I can't fix myself and arealways showing up.

I do wonder: 
* How does this technically happen? I'm sure you're able to help on this :)
* Isn't this feature too much? I mean, I'm not sure that users (if there are) are all used to compile their own file. I still think this is a good idea, I'm just questionning its value for end users.

May another user give his opinion? @GaetanBt or @7studio for example?

**Comment by HugoGiraudel** [comment]
> How does this technically happen? I'm sure you're able to help on this :)

Current:

```scss
@mixin error($key, $self-closing: false) {
  @if is-level-enough("error") {
    @include item("error", $key, $self-closing);
  }
}
```

New: 

```scss
@mixin error($key, $self-closing: false) {
  $is-disabled: index(map-get($disabled-tests, 'error'), $key);

  @if is-level-enough("error") and not $is-disabled {
    @include item("error", $key, $self-closing);
  }
}
```

**Comment by ffoodd** [comment]
Simple enough for me :) 

I think we can implement this right after I fixed a few bugs more.

Thank you Hugo!

**Comment by HugoGiraudel** [comment]
Is a map to update enough, or do we want an API like this to stay consistent?

```scss
@include disable-advices('target-blank', 'empty-class');
@include disable-error('tab-order');
```

**Comment by ffoodd** [comment]
Your new suggestion is probably simpler to use, so I guess it would be better. 

Do you already have this in mind ?

**Comment by HugoGiraudel** [comment]
Done: https://github.com/ffoodd/a11y.css/pull/127.

**Comment by ffoodd** [comment]
Closed with #128 .

Thank you Hugo!
